base/src/main/java/io/spine/validate/NumberFieldValidator.java

```diff
@@ -112,11 +112,21 @@ protected void validateOwnRules() {
         }
     }
 
+    /**
+     * Returns {@code false}.
+     *
+     * @implNote Attempting to define whether a numeric field is set or not is semantically
+     * incorrect, because the way to define whether a value is set or not is to compare it against
+     * its default value, which for numeric fields is {@code 0}, which is sometimes a desired and
+     * valid value for a field.
```

Comment:
This _one_ sentence is too long. Please split it up and simplify. Also, I'm not sure if `<p>` paragraphs are supported in `@implNote`.